### PR TITLE
sci-physics/vgm: new package.

### PR DESCRIPTION
**Pull request by olifre**
VGM (Virtual Geometry Model) is a geometry conversion tool,
actually providing conversion between Geant4 and ROOT TGeo
geometry models.

Package-Manager: Portage-2.3.20, Repoman-2.3.6

This package has previously lived in gentoo/sci (science overlay), since pull requests are starving there, I polished and bumped the package a bit and opened this PR.

### CODE REVIEW

File: sci-physics/vgm/vgm-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+src_test() {
+	cd "${BUILD_DIR}"/test || die
+	./test_suite.sh || die
+}
+
+src_install() {
+	cmake-utils_src_install
+	cd doc || die
+	dodoc README todo.txt VGMhistory.txt VGM.html VGMversions.html
+	use doc && dohtml -r html/*
```
Comment (titanofold) [inline sci-physics/vgm/vgm-9999.ebuild]: `dohtml` is dead.

`dodoc -r html` should do the trick.

**Comment by olifre** [comment]
@titanofold Thanks for noticing!
I took the chance to modernize docs installation to EAPI 6 by using `einstalldocs`.

**Comment by olifre** [comment]
@titanofold Sorry to bump, but since the PR is already approaching page 3 on github and endangered of being lost in old stuff... 
Is there anything still missing which I should take care of?

File: sci-physics/vgm/vgm-9999.ebuild [hunk 2]
```diff
@@ -29,6 +29,14 @@ DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen[dot] )
 	test? ( geant4? ( xml? ( >=sci-physics/geant-4.10.03[gdml] ) ) )"
```
Comment (titanofold) [inline sci-physics/vgm/vgm-9999.ebuild]: Ditto.

File: sci-physics/vgm/vgm-4.4.ebuild [hunk 3]
```diff
@@ -29,6 +29,14 @@ DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen[dot] )
 	test? ( geant4? ( xml? ( >=sci-physics/geant-4.10.03[gdml] ) ) )"
```
Comment (titanofold) [inline sci-physics/vgm/vgm-4.4.ebuild]: The `xml` use flag doesn't make sense. We're not adding XML capabilities to the package, so it's better to just drop `xml`.

**Comment by olifre** [comment]
Well spotted - fixed and squashed!
Many thanks!

**Comment by olifre** [comment]
@titanofold Sorry to bump again - anything still missing?

**Comment by olifre** [comment]
Hey @monsieurp many thanks! :+1:

**Pull request closed without merging**